mock-server/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "freestyle-opscenter-mock-server",
+  "version": "0.0.1",
+  "description": "A simple websocket mock server",
+  "main": "index.js",
+  "scripts": {
+    "start": "node index.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [
+    "server",
+    "websocket",
+    "websockets"
+  ],
+  "author": "Juan Valencia Calvellido",
+  "license": "MIT",
```

Comment:
This should remain the same License as the project ASL 2

Reply:
Addressed in f01481f